posthog/schema.py

```diff
@@ -124,13 +124,10 @@ class ElementType(BaseModel):
 
 
 class EmptyPropertyFilter(BaseModel):
+    pass
     model_config = ConfigDict(
         extra="forbid",
     )
```

Comment:
Hmm.. Will this also skip the `model_config` line then? Worth removing, or there's some magic happening here?